heudiconv/heuristics/reproin.py

```diff
@@ -541,8 +540,14 @@ def infotodict(seqinfo):
 
         # analyze s.protocol_name (series_id is based on it) for full name mapping etc
         if seqtype == 'func' and not seqtype_label:
-            if '_pace_' in series_spec:
+            if s.series_description.endswith('_SBRef'):
+                seqtype_label = 'sbref'
```

Comment:
I am not sure how generalizable this is. Single-band reference scans are automatically outputted in an [X]_SBRef dicom folder with the CMRR MB sequence, where [X] is the name of the multi band scan.

Reply:
note: for me, with whatever sequence we have on our Siemens now (yet to check), `_SBRef` is present only within `SeriesDescription` field, and not `ProtocolName`:
```
$> grep s4p2 000001.txt 
(0008,103e) LO [func_run-01_task-pinel_acq-s4p2-me_SBRef] #  40, 1 SeriesDescription
(0018,1030) LO [func_run-01_task-pinel_acq-s4p2-me]     #  34, 1 ProtocolName
```
and in reproin we "favor" (consider first)  `ProtocolName`:
```shell
$> grep '^series_spec_fields' heudiconv/heuristics/reproin.py
series_spec_fields = ('protocol_name', 'series_description')
```
I have ran into the same situation with other types of data, requiring to switch to use `series_description` in favor of `protocol_name`.  I wonder how safe it would be to switch order generally. Otherwise `reproin` should just become smarter...

Reply:
Should I open an issue about it in the ReproIn repository?

Reply:
nah -- let's solve it HERE!  Sorry for the delay.  Absence of sbref handling in reproin started to bite me, so thank you for leading this effort here! ;)
I will push some tuneups shortly, but here is my question:  When would we get `_phase` images, in particular for "bold"?  Never saw those and do not see BIDS provisioning for them.  BIDS talks about [fmap "Case 2: Two phase images and two magnitude images"](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#case-2-two-phase-images-and-two-magnitude-images)  but that is something we do not get a grip in reproin -- to review multiple scans not independently from each other to decide on indexes (such as `_phase1` and `_phase2` here)

Reply:
You get both `_bold` and `_phase` when you do Magnitude+Phase reconstruction. That's also when you get `rec-magnitude` and `rec-phase` sbrefs (as in #421). Phase is listed as one of the valid task MRI contrasts in the BIDS specification [here](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#task-including-resting-state-imaging-data).

Reply:
THANK YOU!  I was looking for `_phase` and thus didn't hit that BIDS spec link you gave.